File: examples/fortran/cv2018/equivalence/resid/residual_bitand7.c
void RESIDUAL_8(int* N, int  IA1[], int  IA2[]) {
    int I;
    for (I = 1; I <= *N; I = I + 1) {
        IA2[I - 1] = IA1[I - 1] & 7;
    }
    return;
}

File: examples/fortran/cv2018/equivalence/resid/residual_bitand7.f
      SUBROUTINE RESIDUAL_8(N,IA1,IA2)
        INTEGER N,IA1(N),IA2(N)

        Do I=1,N
          IA2(I) = IAND(IA1(I), 7)
        ENDDO
        RETURN
      END
